rules/no-disabled-tests.js

```diff
@@ -1,19 +1,16 @@
 'use strict';
 
 function getName(node) {
-  function joinNames(a, b) {
```

Comment:
@borilla mind sharing why you added the check? None of the tests hit this path, so I just inlined it

Reply:
Just defensive coding, but I can come up with a theoretical example using an anonymous object:
```
({ f: function () {} }).f()
```